src/public/react/src/constants/literals.js

```diff
@@ -37,16 +37,16 @@ export const SIZE = 'Size: ';
 export const DOWNLOAD = 'Download';
 export const PATH_APPLIANCE = '/appliance';
 export const LINK1 =
-  '<a href="http://opennebula.systems/legal-notice">Legal Notice.</a>';
-export const LINK2 = '<a href="mailto:contact@opennebula.org">webmaster.</a>';
+  '<a href="http://opennebula.io/legal-notice">Legal Notice.</a>';
+export const LINK2 = '<a href="mailto:contact@opennebula.io">webmaster.</a>';
 export const LINK3 =
-  '<a href="http://opennebula.systems/">OpenNebula Systems.</a>';
+  '<a href="http://opennebula.io/">OpenNebula Systems.</a>';
 export const FOOTER_TEXT =
-  '© %YEAR% OpenNebula Systems S.L. All Rights Reserved. %LINK1% Please send comments to %LINK2% This site is managed by %LINK3% <br> OpenNebula and OpenNebula Systems are not responsible for any content you may access or download from external sites';
+  '© %YEAR% OpenNebula Systems. All Rights Reserved. %LINK1% Please send comments to %LINK2% This site is managed by %LINK3% <br> OpenNebula and OpenNebula Systems are not responsible for any content you may access or download from external sites';
 export const PRINCIPAL_MENU = [
   {
     title: 'Documentation',
-    url: 'http://marketplace.opennebula.systems/docs/',
+    url: 'http://marketplace.opennebula.io/docs/',
```

Comment:
Better `https://docs.opennebula.io/appliances/`

Also, ideally all the rest references point directly to https://

Reply:
I added the https and the url as you said, thanks!